lib/tasks/gettext.rake

```diff
@@ -90,4 +90,25 @@ namespace :gettext do
       end
     end
   end
+
+  task :extract_locale_names do
```

Comment:
Does this have any prerequisites such as rails environment?  Does `Vmdb::FastGettextHelper.register_locales` need rails loaded?

If so, `task :extract_locale_names => :environment do`